packages/trader/src/Stores/Modules/Trading/Actions/start-date.ts

```diff
@@ -63,7 +56,7 @@ export const onChangeExpiry: TOnChangeExpiry = async store => {
     const obj_market_open_times = { market_open_times: trading_times.open };
     const obj_market_close_times = { market_close_times: trading_times.close };
 
-    const market_close_times = obj_market_close_times.market_close_times;
+    const market_close_times = obj_market_close_times.market_close_times as string[];
```

Comment:
@kate-deriv as string[] is not needed here. please add @ts-expect-error above L55 like i did: https://github.com/kate-deriv/deriv-app/pull/32/files#diff-9be3c492e9632318c2aab3efaf62051bdc1cdb78bb072fedd92d78c73c1b43e5

Reply:
Will try to fix inside of ContractType